PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+#### What's this PR do?
```

Comment:
Where exactly will people be filling and submitting this template? 

I found this "Create a pull request via the github online interface, fill the pull request template, and select the person that will code review your code."

But, where does the template exactly go and how does the other person read it?